src/BroadcastDataController.php

```
@@ -0,0 +1,212 @@
+                    $schedule->getToday()->television
+                ),
+                'tomorrow' => array_map(
+                    fn ($item) => $this->decode($item->name),
+                    $schedule->getTomorrow()->television
+                ),
+            ],
+        ]);
+        $response->header('Cache-Control', 'no-store, no-cache, must-revalidate, max-age=0');
+
+        return $response;
+    }
```

PHPMD (2.15.0)

[error] The method get_item is not named in camelCase. (undefined)

(CamelCaseMethodName)